.gitmodules

```diff
@@ -1,3 +1,6 @@
 [submodule "3rdparty/doctest"]
 	path = 3rdparty/doctest
 	url = https://github.com/onqtam/doctest.git
+[submodule "3rdparty/json"]
```

Comment:
Did you consciously not pick a release for this one? The last one seems to be v3.9.1.

Also, could you update `LICENSE.3rdparty` for the new path?

Reply:
Nope, missed picking a release, will do.